[2025-05-30T01:23:50.229+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-30T01:23:50.351+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flag_check_mysql_dag.execute_task scheduled__2025-05-29T01:50:00+00:00 [queued]>
[2025-05-30T01:23:50.413+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flag_check_mysql_dag.execute_task scheduled__2025-05-29T01:50:00+00:00 [queued]>
[2025-05-30T01:23:50.414+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-05-30T01:23:50.534+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): execute_task> on 2025-05-29 01:50:00+00:00
[2025-05-30T01:23:50.577+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1271) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-30T01:23:50.582+0000] {standard_task_runner.py:72} INFO - Started process 1276 to run task
[2025-05-30T01:23:50.574+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flag_check_mysql_dag', 'execute_task', 'scheduled__2025-05-29T01:50:00+00:00', '--job-id', '166', '--raw', '--subdir', 'DAGS_FOLDER/Sesion3/Ejemplo7.py', '--cfg-path', '/tmp/tmpuia5om83']
[2025-05-30T01:23:50.598+0000] {standard_task_runner.py:105} INFO - Job 166: Subtask execute_task
[2025-05-30T01:23:50.675+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:23:50.859+0000] {task_command.py:467} INFO - Running <TaskInstance: flag_check_mysql_dag.execute_task scheduled__2025-05-29T01:50:00+00:00 [running]> on host 31d54648d280
[2025-05-30T01:23:50.867+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:23:51.294+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flag_check_mysql_dag' AIRFLOW_CTX_TASK_ID='execute_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-29T01:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-29T01:50:00+00:00'
[2025-05-30T01:23:51.298+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-30T01:23:51.471+0000] {logging_mixin.py:190} INFO - Flag detectado, ejecutando tarea...
[2025-05-30T01:23:51.479+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-30T01:23:51.529+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-30T01:23:51.532+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flag_check_mysql_dag, task_id=execute_task, run_id=scheduled__2025-05-29T01:50:00+00:00, execution_date=20250529T015000, start_date=20250530T012350, end_date=20250530T012351
[2025-05-30T01:23:51.704+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-30T01:23:51.759+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
